src/content/pages/social-event.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Thursday Social Event
+subtitle: Thursday Social Event
+---
+
+# Join Us on Střelecký Island for a Relaxed Summer Evening
+
+Right in the heart of Prague, surrounded by the Vltava River and historic
+architecture, Střelecký Island sets the stage for our main social event. Expect
+a laid-back evening with:
+
+- **Beautiful riverside seating** in a green setting just minutes from the city
+  centre—plenty of space to chill.
+- **Board games**, with a dedicated zone for group fun.
+- **Quiet corners** to relax, chat, or simply enjoy the atmosphere.
+- **Live music and jamming**—bring your instruments or join in with others for a
+  jam session!
+
+![Riverside view of Střelecký Island, the social event venue](/images/social-event-venue.jpg)
+
+## Bring Your Board Games
+
+We’ll have space reserved for board games, so whether you're into competitive
+strategy or casual party games, bring your favourites along!
+
+![People playing board and social games at an outdoor event](/images/social-event-games.jpg)
+
+## Bring Your Instruments
+
+Music has been a highlight of the past year, and we’re keeping that tradition
+alive. You’re welcome to bring your own instruments or borrow one from the local
+community. Let’s make some music together!
+
+![People playing music together in an outdoor jam session](/images/social-event-jam.jpg)
+
+## Pack Your Picnic Blankets & Sportswear
+
+Feel like moving around? There will be room to play volleyball, foot tennis,
+pétanque, or croquet. Whether you’re joining in or just watching from your
+blanket, come as you are and enjoy the summer vibe!
+
+## Tickets
+
+**Please note:** This event is not included in the conference ticket.
+
+Tickets will be available soon via our ticket shop and will be limited in
+number. Keep an eye on our announcements on social media and newsletter to make
+sure you don’t miss out.
+
+## When & Where?
+
+- 🗓 **Thursday, July 17th at 7.30 PM** 📍
```

Comment:
I'm not sure how we do this in the rest of the website, but it would be nice if we keep it consistent.

```suggestion
- 🗓 **Thursday, July 17th at 19:30** 📍
```